mindsdb/integrations/handlers/neuralforecast_handler/__about__.py

```diff
@@ -0,0 +1,9 @@
+__title__ = "MindsDB NeuralForecast handler"
+__package_name__ = "mindsdb_neuralforecast_handler"
+__version__ = "0.0.0"
```

Comment:
Definitely 0.0.1 at the very least, by now 😁

Reply:
You're too kind ;)